builders.go

```diff
@@ -88,3 +89,11 @@ func SaramaConsumerBuilderWithConfig(config *sarama.Config) SaramaConsumerBuilde
 		return sarama.NewConsumer(brokers, config)
 	}
 }
+
+// BackoffBuilder creates a backoff
+type BackoffBuilder func() (Backoff, error)
+
+// DefaultBackoffBuilder returnes a simpleBackoff with 10 second steps
+func DefaultBackoffBuilder() (Backoff, error) {
+	return newSimpleBackoff(time.Second * 10), nil
```

Comment:
If we export it like `NewSimpleBackoff`, then users could reuse the backoff and change the step duration without recreating a new backoff strategy.

Reply:
Thats true.